store/sqlstore/scheme_store.go

```diff
@@ -70,18 +69,28 @@ func (s *SqlSchemeStore) Save(scheme *model.Scheme) (*model.Scheme, error) {
 
 	scheme.UpdateAt = model.GetMillis()
 
-	rowsChanged, err := s.GetMaster().Update(scheme)
+	res, err := s.GetMasterX().NamedExec(`UPDATE Schemes
+		SET UpdateAt=:UpdateAt, Name=:Name, DisplayName=:DisplayName, Description=:Description, Scope=:Scope,
```

Comment:
This is missing `CreateAt`, `DeleteAt` as well.

Reply:
Corrected this as well.